tzatziki-spring-jpa/src/test/java/com/decathlon/tzatziki/steps/TestApplicationSteps.java

```diff
@@ -25,15 +25,22 @@
 
     private static final PostgreSQLContainer<?> postgres =
             new PostgreSQLContainer<>("postgres:12").withTmpFs(Map.of("/var/lib/postgresql/data", "rw"));
+    private static final PostgreSQLContainer<?> secondPostgres =
+            new PostgreSQLContainer<>("postgres:12").withTmpFs(Map.of("/var/lib/postgresql/data", "rw"));
 
     static class Initializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
 
         public void initialize(ConfigurableApplicationContext configurableApplicationContext) {
             postgres.start();
+            secondPostgres.start();
             TestPropertyValues.of(
-                    "spring.datasource.url=" + postgres.getJdbcUrl(),
+                    "spring.datasource.jdbc-url=" + postgres.getJdbcUrl(),
```

Comment:
```suggestion
                    "spring.datasource.url=" + postgres.getJdbcUrl(),
```

Ca reste .url pour moi ?

Reply:
.url ça marche pas x)